src/components/Modals/PublishThirdPartyCollectionModal/PublishThirdPartyCollectionModal.tsx

```diff
@@ -3,16 +3,31 @@ import { ModalNavigation, Button } from 'decentraland-ui'
 import Modal from 'decentraland-dapps/dist/containers/Modal'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
 import { SyncStatus } from 'modules/item/types'
-import { isStatusAllowedToPushChanges } from 'modules/item/utils'
+import { isAllowedToPushChanges } from 'modules/item/utils'
 import { PublishButtonAction } from 'components/ThirdPartyCollectionDetailPage/CollectionPublishButton/CollectionPublishButton.types'
 import { getTPButtonActionLabel } from 'components/ThirdPartyCollectionDetailPage/CollectionPublishButton/CollectionPublishButton'
 import { Props } from './PublishThirdPartyCollectionModal.types'
 
 export default class PublishThirdPartyCollectionModal extends React.PureComponent<Props> {
+  getItemsToPublish = () => {
+    const { items, itemsStatus } = this.props
+    return items.filter(item => itemsStatus[item.id] === SyncStatus.UNPUBLISHED)
```

Comment:
I was thinking that maybe this could be a method somewhere else. Either in the `item/selectors.ts` file or in `item/utils.ts`. Mostly because it seems like something we'll use in more than one place, but also because I think it's repeated [here](https://github.com/decentraland/builder/pull/1832/files#r826330790) ? but with a different implementation.

In case the impl HAS to be different, the find for a itemCuration might be a candidate for abstraction as I think is the same [here](https://github.com/decentraland/builder/pull/1829/files#diff-d678e3d56b94dc53c3a5660c2639e82446a8e15df0169c382f04f3e0cf6e7d90R36) too

What do you think?

Reply:
Agreed, it should be outside this component. I moved it to the utils file :) 
I'll update #1832 once this one gets merged!